src/open_prime_rando/echoes_patcher.py

```diff
@@ -97,6 +98,26 @@ def apply_area_modifications(editor: PatcherEditor, configuration: dict[str, dic
             area.build_mlvl_dependencies(only_modified=True)
 
 
+def apply_corrupted_memory_card_change(editor: PatcherEditor):
+    # STRG_MemoryCard_0
+    table = editor.get_file(0x88E242D6, Strg)
+
+    name_to_index = {
+        table.raw.name_table.name_array[entry.offset].string: entry.index
+        for entry in table.raw.name_table.name_entries
+    }
```

Comment:
this should be part of the RDS API for Strg probably

Reply:
Absolutely. I just uhh dodged needing yet another rds to get this working :p